netlify.toml

```diff
@@ -25,6 +25,11 @@
     Content-Security-Policy = "default-src 'self'; connect-src 'self' analytics.google.com stats.g.doubleclick.net; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com 'unsafe-eval' media-library.cloudinary.com; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
     Permissions-Policy = "camera=(), microphone=(), geolocation=(), interest-cohort=()"
 
+[[headers]]
+  for = "/service-worker.js"
+  [headers.values]
+    Cache-Control = "no-store, max-age=0"
```

Comment:
Don't forget to duplicate this header into `next.config.js` as well, just in case `@netlify/plugin-nextjs` supports headers/rewrites directly from `next.config.js` in the near future.

Reply:
got it